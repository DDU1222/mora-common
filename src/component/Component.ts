import * as React from 'react'

export default class Component<P, S> extends React.Component<P, S> {

}

