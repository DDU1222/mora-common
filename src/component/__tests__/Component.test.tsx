import * as React from 'react'
import Component from '../Component'
import * as renderer from 'react-test-renderer'

class A extends Component<any> {
  render() {
    return <div>aa</div>
  }
}

test('react', () => {
  let component = renderer.create(<A />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
