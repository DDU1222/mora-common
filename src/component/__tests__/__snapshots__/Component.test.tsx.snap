// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react 1`] = `
<div>
  aa
</div>
`;
